src/hotspot/share/gc/g1/g1CollectedHeap.hpp

```diff
@@ -779,7 +779,19 @@ class G1CollectedHeap : public CollectedHeap {
 
   G1MonotonicArenaFreePool _card_set_freelist_pool;
 
+  // Group cardsets
+  G1CardSetMemoryManager _card_set_mm;
```

Comment:
Is it possible to rename this to `_young_regions_card_set_mm` to make it more clear this is the card set memory manager for young regions?